marked-element.html

```diff
@@ -84,6 +84,38 @@
       padding-left: 24px;
     }
 
+
+### Syntax Highlighting
+Provides an opportunity to implement syntax highlighting, [e.g with highlightjs](https://highlightjs.org/).
+An example handler can have this implementation : 
+```
+<marked-element id="syntaxHighlight" on-syntax-highlight="_highlightSyntax">
+  ...
+</marked-element>
+```
+
+```
+_highlightSyntax(e, detail) {
+   detail.code = hljs.highlightAuto(detail.code).value;
+   return e;
+ }
+```
+If using highlightjs, ensure the javascript library is loaded. You might also
```

Comment:
Can this be reworded for a generic library? e.g.

If using a highlighting library, ensure that it is loaded prior to (attaching `marked-element`? Not exactly sure when tbh). It may also be necessary to inline (can you please expand a bit on what you mean by _inline_) the specific style you intend to use...

E.g. (then you give a specific highlightjs example)